esp/esp/customforms/DynamicModel.py

```diff
@@ -217,25 +218,33 @@ def get_field_name(self, field):
     def addField(self, field):
```

Comment:
So I was able to add literally every field to a form and save it, but when I go to fill it out, I get this:

![image](https://user-images.githubusercontent.com/25392075/118063958-1d088380-b368-11eb-9aa4-4963d0c011b3.png)

Reply:
If you need me to add fields one by one to debug if there's a certain field causing the error, but I'd rather not if you can spot the issue without that :P

Reply:
Ah, I think what's going on here is that the "correct answers" are supposed to be the indices of the choices, not the text values of the choices? Could you try that and see if it works?

Reply:
yeah, i did 1-based indices. Switching to 0 based fixed the form. 
Can we do at least one of the following:
- specify that 0 based indexing is desired
- use 1 based indexing here because 0 based in a non-coding setting is unusual
- label the choices with their index (1. ____ 2. ____ etc)
- check that the specified correct answer is valid before creating the form